firebase-sessions/src/main/kotlin/com/google/firebase/sessions/FirebaseSessions.kt

```diff
@@ -35,11 +35,11 @@ internal constructor(
   backgroundDispatcher: CoroutineDispatcher,
   transportFactoryProvider: Provider<TransportFactory>,
 ) {
-  private val sessionGenerator = SessionGenerator(collectEvents = true)
+  private val sessionSettings = SessionsSettings(firebaseApp.applicationContext)
```

Comment:
This is good. But we need to find out a way to wait for some seconds before initiating a network config fetch in the settings layer to fetch configs.

Reply:
Not enough of settings is implemented yet to do this now. I will keep this in mind though.